src/styles/blocks/header.scss

```
@@ -1,42 +1,29 @@
-.header {
+&__list {
   display: flex;
-  align-items: center;
-  justify-content: space-between;
-  box-sizing: border-box;
-
-  height: 60px;
-  padding: 0 50px;
-
-  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
-
-  &__logo {
-    height: 40px;
-  }
+  margin: 0;
+  padding: 0;
 }
 
-.nav {
-  &__list {
-    display: flex;
-    margin: 0;
-    padding: 0;
+&__item {
+  position: relative;
+  color: #060b35;
+  list-style: none;
+  line-height: 60px;
+  &:not(:last-child) {
+    margin-right: 20px;
   }
+}
 
-  &__item {
-    position: relative;
-    color: #060b35;
-    list-style: none;
-    line-height: 60px;
+&__link {
+  color: inherit;
+  text-decoration: none;
+  text-transform: uppercase;
+  font-size: 12px;
+  font-weight: 500;
 
-    &:not(:last-child) {
-      margin-right: 20px;
-    }
-  }
+  transition: color, $transitionDuration;
```

This is incorrect, comma should separate the transitions group
```suggestion
  transition: color $transitionDuration;
```